var/spack/repos/spack_repo/builtin/packages/gaudi/package.py

```diff
@@ -87,6 +87,15 @@ class Gaudi(CMakePackage, CudaPackage):
     # add a few missing includes (c++20?)
     patch("includes.patch", when="@37:38")
 
+    # Remove vestigial ROOT component requirement (redundant with
+    # v34r1, conflicts with ROOT >=6.36). See
+    # https://gitlab.cern.ch/gaudi/Gaudi/-/commit/89f38fab9052093de98d06a2a6e5fda2c9574007
+    patch(
+        "https://gitlab.cern.ch/gaudi/Gaudi/-/commit/89f38fab9052093de98d06a2a6e5fda2c9574007.diff",
+        sha256="6b377fd10828bf26367c26792a5465351f3f0b5f7f6073dbcae6fa9195d4a414",
+        when="@34.1:39",
```

Comment:
Patch does not apply to 34.1 since `GaudiTestSuite/CMakeLists.txt` only exists after 38.1 due to https://gitlab.cern.ch/gaudi/Gaudi/-/commit/b85e225f8d2020608382168982dd635b6ca6a353. Checked that it applies cleanly to 38.1.

```suggestion
        when="@38.1:39",
```